### PR TITLE
Remove function tracer

### PR DESCRIPTION
**Pull request by jkrocil**
Purpose or Intent
=================
Not used anymore; prints message to stdout when `--tracer` or `--no-tracer` is used (for backward compatibility) but doesn't do anything anymore.

```
# py.test --no-tracer cfme/tests --collect-only
The --no-tracer option has been REMOVED, it doesn't do anything; please, stop using it!
Loading templates from trackerbot
...
```

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8f4555e89dad991e314ded835b709ea5cdc8b8fe:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d1565ac28229c9f6204f5b8efe6b0c08133f0818:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: conftest.py [hunk 1]
```diff
@@ -34,6 +34,18 @@ def __str__(self):
...
+    # Keeping due to compatibility
+    parser.addoption("--no-tracer", dest="no_tracer", action="store_true", default=False,
+                     help="Disable the function tracer (REMOVED, doesn't do anything!)")
+    parser.addoption("--tracer", dest="tracer", action="store_true", default=False,
+                     help="Enable the function tracer (REMOVED, doesn't do anything!)")
+
+
+def pytest_cmdline_main(config):
+    for opt in ['--tracer', '--no-tracer']:
+        if config.getoption(opt) is True:
```
Comment (RonnyPfannschmidt) [inline conftest.py]: i think the is is not needed

File: conftest.py [hunk 2]
```diff
@@ -34,6 +34,18 @@ def __str__(self):
...
     parser.getgroup('cfme', 'cfme: options related to cfme/miq appliances')
+    # Keeping due to compatibility
+    parser.addoption("--no-tracer", dest="no_tracer", action="store_true", default=False,
+                     help="Disable the function tracer (REMOVED, doesn't do anything!)")
+    parser.addoption("--tracer", dest="tracer", action="store_true", default=False,
+                     help="Enable the function tracer (REMOVED, doesn't do anything!)")
+
+
+def pytest_cmdline_main(config):
+    for opt in ['--tracer', '--no-tracer']:
```
Comment (RonnyPfannschmidt) [inline conftest.py]: how about using config.option.tracer or config.option.no_tracer

alternatively i think you can use the combination of meta and using the same destination to just use a single `deprecated_tracer_arg_used` option

File: conftest.py (hunk 1 above)
Comment (RonnyPfannschmidt) [inline conftest.py]: @jkrocil in general its considered bad to explicitly check for the boolean value in boolean context,

i think more recent pyflakes/flake8 even warns on it

**Comment by dajoRH** [comment]
Lint report for commit cba10b2e8016e756c30af5e49824e4067060c346:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: conftest.py (hunk 1 above)
Comment (jkrocil) [inline conftest.py]: Done.

**Pull request merged**